<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSwingTestBinarySignalDlg</class>
 <widget class="QDialog" name="QSwingTestBinarySignalDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>209</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QSttCheckBox" name="m_cbBout1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>开出1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSttCheckBox" name="m_cbBout5">
       <property name="text">
        <string>开出5</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSttCheckBox" name="m_cbBout2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>开出2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSttCheckBox" name="m_cbBout3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>开出3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSttCheckBox" name="m_cbBout4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>开出4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSttCheckBox" name="m_cbBout6">
       <property name="text">
        <string>开出6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSttCheckBox" name="m_cbBout8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>开出8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSttCheckBox" name="m_cbBout7">
       <property name="text">
        <string>开出7</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_22">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnOk">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_23">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSttCheckBox</class>
   <extends>QCheckBox</extends>
   <header>../Controls/SttCheckBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
